client/src/app/map/map.component.ts

```diff
@@ -404,45 +404,14 @@ export class MapComponent implements OnInit {
                     attributions: [new ol.Attribution({html: layerData.layerCopyright})],
                 })
             });
-            if (layerData.layerName === 'earthnetworks_lightning_points'){
-                layer.setStyle(this.styleService.getLightningStyle)
-            }
-            else if (layerData.layerName === 'ktm_pcdm_at_risk_village'){
-                layer.setStyle(this.styleService.getAtRiskVillageStyle);
-            }
-            else if (layerData.layerName === 'ktm_pcdm_at_risk_commune') {
-                layer.setStyle(this.styleService.getAtRiskCommuneStyle);
-            }
-            else if (layerData.layerName === 'wfp_daily_people_affected'){
-                layer.setStyle(this.styleService.getDailyPeopleAffectedStyle);
-            }
-            else if (layerData.layerName === 'wfp_daily_displaced'){
-                layer.setStyle(this.styleService.getDailyDisplacedStyle);
-            }
-            else if (layerData.layerName === 'wfp_daily_deaths'){
-                layer.setStyle(this.styleService.getDailyDeathsStyle);
-            }
-            else if (layerData.layerName === 'wfp_daily_pumpwells'){
-                layer.setStyle(this.styleService.getDailyPumpWells);
-            }
-            else if (layerData.layerName === 'wfp_daily_healthcenter'){
-                layer.setStyle(this.styleService.getDailyHealthCenter);
-            }
-            else if (layerData.layerName === 'wfp_daily_school'){
-                layer.setStyle(this.styleService.getDailySchool);
-            }
-            else if (layerData.layerName === 'wfp_daily_road'){
-                layer.setStyle(this.styleService.getDailyRoad);
-            }
-            else if (layerData.layerName === 'wfp_daily_bridge'){
-                layer.setStyle(this.styleService.getDailyBridge);
-            }
-            else if (layerData.layerName === 'wfp_daily_rice'){
-                layer.setStyle(this.styleService.getDailyRice);
-            }
-            else {
-                layer.setStyle(this.styleService.getStyle);
+            var features = layer.getSource().getFeatures();
+            for (var i = 0; i < features.length; i++){
+                features[i].setProperties({
+                    layerStyle: layerData.layerStyle
```

Comment:
Consider setting style object (calculated by the function) rather than a function if we need to apply this to every featue anyway

Reply:
Done